cmd/clusterctl/examples/openstack/machines.yaml.template

```diff
@@ -19,8 +19,8 @@ items:
         securityGroups:
         - default
     versions:
-      kubelet: 1.9.4
-      controlPlane: 1.9.4
+      kubelet: 1.12.1
```

Comment:
Just wanted to note that 1.12 is only required in the bootstrap cluster, not in any of the child clusters ;)